### PR TITLE
Remove user details from sn header

### PR DESCRIPTION
**Pull request by jon-stevens**
I have removed references to the user-details component, as this has been replaced with the Identity component.
I moved some styles from core.scss to enhanced.scss.
I moved top padding from .c-header to .c-header__inner.
I tidied some of the rules around sizing for the logos.

### CODE REVIEW

File: toolkits/springernature/packages/springernature-header/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 3.1.0 (2022-12-06)
```
Comment (tulipdexter) [inline toolkits/springernature/packages/springernature-header/HISTORY.md:3]: Happy new year Jon!

Comment (tulipdexter) [reply toolkits/springernature/packages/springernature-header/HISTORY.md:3]: Wait, you did this in December. I'm awake now

**Review by tulipdexter — APPROVED** [review]
LGTM 👍

File: toolkits/springernature/packages/springernature-header/HISTORY.md (hunk 1 above)
Comment (jon-stevens) [reply toolkits/springernature/packages/springernature-header/HISTORY.md:3]: > Wait, you did this in December. I'm awake now

Yeah I think it went a bit under the radar end of December

**Pull request merged**